### ISSUE TITLE
Flight services should not be displayed for past trips

### ISSUE THREAD

**Issue by tbergq** [kiwicom/mobile#670]

**Comment by mrtnzlml** [comment]
And it should not be displayed for trips with only trains/busses as well.

**Comment by tbergq** [comment]
> And it should not be displayed for trips with only trains/busses as well.

Yeah, we need a test booking for this. I will create a separate issue for it

**Comment by mrtnzlml** [comment]
@tbergq Did you request new booking already?

**Comment by tbergq** [comment]
No, who would help us with this, in the past, I asked @MRTNKCR, but he is on vacation. 🤷‍♂️

**Comment by mrtnzlml** [comment]
@tbergq I just asked on Slack.
